package.json

```diff
@@ -38,17 +38,17 @@
     "grunt": "^0.4.5",
     "grunt-banner": "^0.6.0",
     "grunt-browserify": "^4.0.1",
-    "grunt-contrib-clean": "^0.7.0",
-    "grunt-contrib-jshint": "^0.11.0",
-    "grunt-contrib-nodeunit": "^0.4.1",
-    "grunt-contrib-qunit": "^0.7.0",
+    "grunt-contrib-clean": "^1.0.0",
+    "grunt-contrib-jshint": "^1.0.0",
+    "grunt-contrib-nodeunit": "^1.0.0",
+    "grunt-contrib-qunit": "^1.0.1",
     "grunt-contrib-watch": "^0.6.1",
-    "grunt-eslint": "^10.0.0",
+    "grunt-eslint": "^18.0.0",
     "grunt-exec": "^0.4.6",
-    "grunt-jscs": "^1.5.0",
-    "jquery": "~2.1.3",
+    "grunt-jscs": "^2.8.0",
+    "jquery": "~2.2.1",
```

Comment:
OK I'm gonna change it then. Let me know if there are any issues with the pr
On Mar 12, 2016 12:13, "Chris Rebert" <notifications@github.com> wrote:

> In package.json
> <https://github.com/twbs/bootlint/pull/360#discussion_r55919500>:
>
> >      "grunt-exec": "^0.4.6",
> > -    "grunt-jscs": "^1.5.0",
> > -    "jquery": "~2.1.3",
> > +    "grunt-jscs": "^2.8.0",
> > +    "jquery": "~2.2.1",
>
> Not really
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/twbs/bootlint/pull/360/files#r55919500>.
>